database/seed.js

```diff
@@ -17,15 +17,17 @@ const seedBundles = () => {
       bundleId: bundleId,
       tierId: tierId1,
       tierOrder: 1,
+      cost: cost,
     }, {
       bundleId: bundleId,
       tierId: tierId2,
-      cost: cost,
       tierOrder: 2,
+      cost: cost,
```

Comment:
Just using these lines as an example, look also above and below. Is there a reason you're opting not to use ES6 here? I.e., not using:
`{`
`  bundleId,`
`  cost,`
`  tierId: tierId2,`
`  tierOrder: 2,`
`}`

Reply:
Since some could use the same id & some couldn't, I decided to keep them consistent for better readability.